### PR TITLE
Feature/ds4 18 move agv to assembly

### PR DESCRIPTION
**Pull request by 3FCO**
Depends on #9

### CODE REVIEW

File: backend/CommonAGV/src/main/java/dk/sdu/controllab/commonagv/AGVStatusResponse.java [hunk 1]
```diff
@@ -14,4 +15,10 @@ public AGVStatusResponse(int battery, String programName, AGVState state) {
         this.programName = programName;
```
Comment (Siole) [inline backend/CommonAGV/src/main/java/dk/sdu/controllab/commonagv/AGVStatusResponse.java:15]: Any reason for leaving out battery in the constructor?

**Review by Siole — CHANGES_REQUESTED** [review]
See comment

File: backend/CommonAGV/src/main/java/dk/sdu/controllab/commonagv/AGVStatusResponse.java (hunk 1 above)
Comment (3FCO) [reply backend/CommonAGV/src/main/java/dk/sdu/controllab/commonagv/AGVStatusResponse.java:15]: ``this.battery = battery;`` on line 14?

Comment (Siole) [reply backend/CommonAGV/src/main/java/dk/sdu/controllab/commonagv/AGVStatusResponse.java:15]: yes

**Review by Siole — APPROVED** [review]

**Pull request merged**